databroker/src/broker.rs

```diff
@@ -728,6 +787,19 @@ impl ChangeSubscription {
                                             }
                                             // fill unit field always
                                             update.unit.clone_from(&entry.metadata.unit);
+                                            update.description = Some(entry.metadata.description.clone());
+                                            #[cfg(feature = "otel")] // This block will only compile if the "otel" feature is enabled
+                                            {
+                                            let mut metadata = MetadataMap::new();
+                                            //  @TODO: Speak to Kuksa team regarding MetadataMap in proto file
+                                            let mut injector = MetadataMapInjector(&mut metadata);
+                                            opentelemetry::global::get_text_map_propagator(|propagator| {
+                                                propagator.inject_context(&current_span.context(), &mut injector);
+                                            });
+                                            let description = metadatamap_to_string(&metadata);
+
+                                            update.description = Some(description);
```

Comment:
please do not missuse description. Description is something to describe a datapoint.